packages/dashmate/configs/getConfigFileMigrationsFactory.js

```diff
@@ -347,6 +347,14 @@ export default function getConfigFileMigrationsFactory(homeDir, defaultConfigs)
 
         return configFile;
       },
+      '0.25.16-rc.6': (configFile) => {
+        Object.entries(configFile.configs)
+          .forEach(([name, options]) => {
+            options.core.docker.image = base.get('core.docker.image');
```

Comment:
testnet still on old version. Please clarify what version is with Latte

Reply:
It's okay, and must be updated